CONTRIBUTORS.yaml

```diff
@@ -12,825 +12,829 @@
 #
 # collection names should be equal to github username, if not, add github: false tag
 ---
-
 # two dummy entries so that autogenerated tutorials do not fail to build
 contributor1:
-    name: Contributor A (Please add contributors to the CONTRIBUTORS.yaml file)
```

Comment:
I am guessing your text editor did this automatically, but please undo all the indentation changes here

Reply:
Yes, I believe my editor auto-formatted it. I will undo the changes. Sorry for the inconvenience.